packages/go/dawgs/cardinality/cardinality.go

```diff
@@ -72,7 +72,7 @@ type Duplex[T uint32 | uint64] interface {
 	Remove(value T)
 	Slice() []T
 	Contains(value T) bool
-	Each(delegate func(value T) (bool, error)) error
```

Comment:
Changing this interface since it was almost never* used with an actual error condition. Turns out, being void with a bool return was how it was primarily being used already

*There originally was one case where it could simply be changed to logging the error and returning false. There was another added between the time this change was made and review time, which I added a closureErr for.

Reply:
Positive change :+1: